capabilities/writetarget/write_target_monitor.go

```diff
@@ -0,0 +1,219 @@
+package writetarget
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/write_target/monitor"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/write_target/pb/data-feeds/on-chain/registry"
+	wt "github.com/smartcontractkit/chainlink-common/pkg/capabilities/write_target/pb/platform"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/write_target/pb/platform/on-chain/forwarder"
```

Comment:
Why are these in the chainlink-common module vs. here within this module?

Reply:
No particular reason, I think that's just what we decided when we first discussed where things should be extracted to, plus, some of the extracted code is product agnostic / not related to WT. But the code in `capabilities/write_target` in common should probably be moved here.

Reply:
Just moved the everything WT specific from common to this repo

Reply:
Think of WT framework similar to TXM framework - a team is going to own it, on a layer that is not global.

chainlink-common is a global bucket, and we should limit what we publish there.